paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -163,6 +164,8 @@ internal fun Wallet(
         }
 
         state.link?.let { link ->
+            Spacer(modifier = Modifier.requiredHeight(8.dp))
```

Comment:
Does this mess up if google pay is not enabled?

Reply:
Oh yeah, it would. Will do a quick fix now and then revisit when we’re taking a more systematic approach to enforcing consistent spacing.